src/poliastro/twobody/orbit.py

```diff
@@ -237,6 +238,15 @@ def __str__(self):
     def __repr__(self):
         return self.__str__()
 
+    def set_true_anomaly(self, target_nu):
```

Comment:
I don't like this approach very much. On the one hand, the `set_*` seems to suggest that we are changing the value of an element _in place_, which we don't do. On the other hand, seems a bit heavy to do all the elements conversion to change only one value.